src/users/users.service.ts

```diff
@@ -78,14 +115,23 @@ export class UsersService {
                 user.company = company
             }
         }
-
         user.employeeId = createuserDto.employeeId;
         user.companyId = createuserDto.companyId;
 
         user.firstName = createuserDto.firstName;
         user.lastName = createuserDto.lastName;
 
+        // Will add different level of admin 
         user.isManager = Boolean(createuserDto.isManager);
+        if (createuserDto.role === Role.Admin){
```

Comment:
In order to future proof this I would make the if statement check if a user role is specified rather than checking if it is Admin. We might add more roles in the future so we don't wan't to have to come back here and change this stuff.

Reply:
`// Will add different level of admin 
        user.isManager = Boolean(createuserDto.isManager);
        if (createuserDto.role === Role.Admin){
            user.role = createuserDto.role;
        }
        // else {
        //     if (user.isManager) {
        //         user.role = Role.Admin
        //     }
        // }`

I think you are mentioning the older commit. The lastest one assigns role based on createuserdto object.

Reply:
`if (createuserDto.role === Role.Admin){` is this line in an older commit?

I meant that instead of writing this: `if (createuserDto.role === Role.Admin){` we do something like this: `if (createuserDto.role){`

Reply:
Oh ok, my bad